test/token/ERC20/ERC20.test.js

```diff
@@ -287,6 +287,46 @@ contract('ERC20', function (accounts) {
     });
   });
 
+  describe('_update', function () {
+    const amount = initialSupply + 1;
+    it('from is the zero address', async function () {
+      const totalSupply = await this.token.totalSupply();
+      const newSupply = totalSupply.add(new BN(amount));
+      await this.token.update(ZERO_ADDRESS, recipient, amount);
+      
+      expect(await this.token.totalSupply()).to.be.bignumber.equal(newSupply);
+      expect(await this.token.balanceOf(recipient)).to.be.bignumber.equal(amount);
+    });
+
+    it('to is the zero address', async function () {
+      const totalSupply = await this.token.totalSupply();
+      const burnAmount = await this.token.balanceOf(initialHolder);
+      await this.token.update(initialHolder, ZERO_ADDRESS, burnAmount);
+      
+      expect(await this.token.totalSupply()).to.be.bignumber.equal(totalSupply.sub(burnAmount));
+      expect(await this.token.balanceOf(initialHolder)).to.be.bignumber.equal(new BN(0));
+    }); 
+
+    describe('from and to are the zero address', function () {
```

Comment:
It's odd that this is tested differently to the other two cases.

That said, I'm not sure that from and to zero addresses should be a valid function call. It sounds to me like it should revert.

Reply:
> That said, I'm not sure that from and to zero addresses should be a valid function call. It sounds to me like it should revert.

IMO this is a no-op, and we always said that no-op should no revert. Also the whole point of this PR is to fix the behavior when this is called.